### ISSUE TITLE
Expose WriteMode in JAPI.

### ISSUE THREAD

**Issue by rmetzger** [stratosphere/stratosphere#781]
I just got a mail from a user, asking how to set the Write mode for csv output.
Currently, you have to do the following:

``` java
CsvOutputFormat<ShippingPriorityItem> csvOut = 
    new CsvOutputFormat<ShippingPriorityItem>(new Path(resultPath), "\n", "|");
csvOut.setWriteMode(WriteMode.OVERWRITE);
joined.output(csvOut);
```

Do you think we should add another variant of `writeAsCsv()` with the write mode?

**Comment by StephanEwen** [comment]
Should be easy to add. Do you (or the user) want to make a pull request for
this?

**Comment by rmetzger** [comment]
Sure, I can add it. (The question was, if we want this in general or not. But I think that somebody asked answers already the question.

**Comment by fhueske** [comment]
Fixed with PR #797
